include/posts.php

```diff
@@ -0,0 +1,32 @@
+<?php
+function blogData () {
+    $posts = array(
+        0 => array(
+            'postID' => 0,
+            'header' => 'what im doing',
+            'title' => 'what im doing',
+            'body' => 'Im currently studying psychology and economics and Im hungry',
+            'nextPost' => 1
```

Comment:
This is an interesting addition. Not a bad idea, but when you're back, let's talk through how you see this working as you build out your site. I might be able to suggest an approach that will scale better.